### ISSUE TITLE
Conan build profile doesn't exist

### ISSUE THREAD

**Issue by gapuchi** [cryfs/homebrew-tap#14]
I'm encountering an issue when installing cryfs on my M1 Mac

```
$ brew install cryfs/tap/cryfs
```

```
-- Conan executing: /opt/homebrew/bin/conan install /tmp/cryfs-20230417-17256-9mkzb2/conanfile.py --build missing --settings build_type=Release --settings compiler=apple-clang --settings compiler.version=14.0 --settings compiler.libcxx=libc++
ERROR: The default build profile '/private/tmp/cryfs-20230417-17256-9mkzb2/.brew_home/.conan2/profiles/default' doesn't exist.
You need to create a default profile (type 'conan profile detect' command)
or specify your own profile with '--profile:build=<myprofile>'
CMake Error at cmake-utils/conan.cmake:633 (message):
  Conan install failed='1'
Call Stack (most recent call first):
  cmake-utils/DependenciesFromConan.cmake:4 (conan_cmake_install)
  CMakeLists.txt:47 (include)
```

Running `conan profile detect` doesn't help since it seems to need a profile relative to CryFs

**Comment by owais** [comment]
Running into the same issue. Any workarounds @gapuchi ?

**Comment by atanaschristov** [comment]
I am having the same issue as well

**Comment by PhilippSoeder** [comment]
same here

**Comment by smessmer** [comment]
This happened because conan2 was released and homebrew then tried to use conan2 for building cryfs, but cryfs doesn't work with conan2 yet. I'm waiting for [the cmake integration of conan 2](https://github.com/conan-io/cmake-conan) to be stable before migrating to conan2.

But I was able to change the cryfs formula so that it forces homebrew to use conan1 for building cryfs and ignore whichever conan version may be available on the system. https://github.com/cryfs/homebrew-tap/pull/15 should fix this issue.
